lib/sisimai/bite/email.rb

```diff
@@ -0,0 +1,53 @@
+module Sisimai
+  module Bite
+    module Email
+      # Sisimai::Bite::Email- Base class for Sisimai::Bite::Email::*
+      class << self
+        # Imported from p5-Sisimail/lib/Sisimai/Bite/Email.pm
+        require 'sisimai/bite'
+        require 'sisimai/rfc5322'
+
+        # @abstract Flags for position variable
+        # @return   [Hash] Position flag data
+        # @private
+        def INDICATORS
+          return {
+            :'deliverystatus' => (1 << 1),
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.

<!-- Comment by [SideCI](https://sideci.com) -->